app/controllers/claims_controller.rb

```diff
@@ -11,7 +11,7 @@ def new
     end
     render locals: {
       contribution: contribution,
-      email: current_user.person&.email
+      email: current_user.email
     }
   end
 
```

Comment:
the create redirect_to goes to the wrong url, but we'll handle that in another PR/ticket